services/brain_failover.py

```
@@ -390,6 +390,33 @@ def is_healthy(self) -> bool:
+# ``models`` (the failover chain).
+try:
+    from packages.ai.brain_config import PROVIDER_CANDIDATES as _CATALOG_CANDIDATES  # noqa: E402
+
+    for _entry in _PROVIDER_REGISTRY:
+        _pid = _entry.get("id")
+        _cands = _CATALOG_CANDIDATES.get(_pid)
+        if _cands:
+            _entry["default_model"] = _cands[0]
+            _entry["models"] = list(_cands)
+except Exception:  # noqa: BLE001 — never break module import
+    pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)